README.md

```diff
@@ -1 +1,3 @@
-# monsterInMe
\ No newline at end of file
+# monsterInMe
+#DEMO </br>
+<img src="img/gif/demo.gif" alt="Demo GIF" border="0">
```

Comment:
Don't add html tags here. Please use markdown.

Reply:
Done! Removed HTML tag and added by simply pressing enter button!

Reply:
I meant this image tag. There is markdown syntax to add an image.